modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneConfigurationParameters.java

```diff
@@ -41,6 +43,9 @@ public class DistributionZoneConfigurationParameters {
     /** Number of zone partitions. */
     private final Integer partitions;
```

Comment:
This class is a simple container for delivering the zone configuration params from DDL to lower level. If any params is null, it will be replaced by defaults here org.apache.ignite.internal.distributionzones.DistributionZoneManager#createZone